- warning: format specifies type 'long' but the argument has type 'time_t' (aka 'long long')

Index: src/develop/masks.h
--- src/develop/masks.h.orig
+++ src/develop/masks.h
@@ -983,7 +983,7 @@ void dt_masks_dynbuf_debug_print(dt_masks_dynbuf_t *a,
   {
     FILE *f;
     char filename[255] = { 0 };
-    sprintf(filename, "debug-%ld-%s", time(NULL), a->tag);
+    sprintf(filename, "debug-%" PRIdMAX "-%s", (intmax_t)time(NULL), a->tag);
     f = g_fopen(filename, "w");
     for (size_t i = 0; i < a->pos; i += 2)
     {
